From: Markus Koschany <apo@debian.org>
Date: Thu, 4 Nov 2021 22:07:37 +0100
Subject: remove junit-bom dependency

Remove junit-bom dependency of scope import which is superfluous and leads to
FTBFS in reverse-dependencies.

See #998606 for example.

Forwarded: not-needed
---
 pom.xml | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/pom.xml b/pom.xml
index d3372ca..86080e5 100644
--- a/pom.xml
+++ b/pom.xml
@@ -237,18 +237,6 @@ file comparators, endian transformation classes, and much more.
     </contributor>
   </contributors>
 
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>org.junit</groupId>
-        <artifactId>junit-bom</artifactId>
-        <version>5.7.2</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
-  
   <dependencies>
     <dependency>
       <groupId>org.junit.jupiter</groupId>
